test/JDBC/expected/sys-all_sql_modules-vu-verify.out

```diff
@@ -1,6 +1,8 @@
+USE sys_all_sql_modules_vu_prepare_db
+GO
+
 -- Test for function
 SELECT
-    definition,
```

Comment:
Can you please explain reason for this?

Reply:
The definition changes between upgrade because if the database didn't upgrade, the definition would produce `CREATE OR REPLACE dbo.OBJECT_NAME`, however after an upgrade, it changes to `CREATE OR REPLACE sys_all_sql_modules_vu_prepare_db_dbo.FUNCTION_NAME`.  This causes an issue when we do JDBC test vs latest->latest upgrade test. From my understanding, we can't have a specific `latest` prepare input and output that differ from the JDBC test.

One thing to note is that the way this view retrieves function and procedure definition is not correct as it uses Pgpsql dialect and uses postgres' built-in function `pg_get_functiondef()`. This was in the original code before any SSMS work and we have left it in. We also can't change the behaviour of this function since it is from postgres itself and not a custom function. 

But because the definition of this is not using a proper custom method to retrieve function or proc definitions using TSQL's syntax, I propose to remove the definition for the latest update unless you have a better suggestion? I can also comment (and create if necessary) some jira tickets for retrieving function defs in TSQL and to add it back to all the sys.sql_modules views once it has been created?

Reply:
This is fixed/resolved after using tsql_view_definition() from Sholini's suggestion